relay-dynamic-config/src/metrics.rs

```diff
@@ -638,6 +643,38 @@ pub fn convert_conditional_tagging(project_config: &mut ProjectConfig) {
     }
 }
 
+/// TODO(ja): Doc
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct ExtrapolateConfig {
```

Comment:
```suggestion
pub struct ExtrapolationConfig {
```

Using the noun here matches what we do elsewhere e.g. `Processing*` not `Process*`